doc/geany.txt

```diff
@@ -4616,15 +4624,15 @@ marker_mark
     *Example:* ``marker_mark=0x000000;0xb8f4b8;false;false``
 
 marker_translucency
-    Translucency for the line marker (first argument) and the search marker
-    (second argument). Values between 0 and 256 are accepted.
+    Translucency for the line marker (first argument) and the mark marker
```

Comment:
the other documentation (on <kbd>^M</kbd>, <kbd>^.</kbd> and <kbd>^,</kbd>) calls those "markers" only, and some other parts refers to them as "bookmarks".